packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```diff
@@ -75,6 +78,8 @@ Summary: NVIDIA %{tesla_major} Open GPU driver
 Version: %{tesla_ver}
 License: MIT AND GPL-2.0-only
 Requires: %{_cross_os}variant-platform(aws)
+Requires: %{name}
+Requires: %{name}-tesla
```

Comment:
Why this instead of just the requirement on the parent package?
```suggestion
```

Reply:
Good catch, the intention was to just depend on the parent package.

Reply:
Fixed!